test/re_test.cc

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 #include <gtest/gtest.h>
-#include "re.h"
+#include "../src/re.h"
```

Comment:
oh i get that. the re stuff has been a bit of a weird extra thing that we test -- i wonder if there's a better way of managing it.

Maybe we should add tests to the benchmark suite to explicitly test the 'find matching benchmarks' functionality instead, which is part of the public API.